service/src/java/org/apache/hive/service/cli/operation/SQLOperation.java

```diff
@@ -200,6 +198,8 @@ public void run() {
           .toTHandleIdentifier().getGuid()).trim();
       driver.setOperationId(guid64);
 
+      driver.setPublicOperationId(getHandle().getHandleIdentifier().getPublicId().toString());
```

Comment:
Are we sure sqlOpDisplay.getOperationId() does not have it?

Reply:
They resolve to the same value. Good catch